packages/manager/apps/freefax/src/controller.js

```diff
@@ -0,0 +1,96 @@
+import { emit, listen } from '@ovh-ux/ufrontend/communication';
+import { Environment } from '@ovh-ux/manager-config';
+
+export default class FreefaxAppController {
```

Comment:
Do you think this needs to be added to the sao generator ?

Reply:
Yes it could be. For now it doesn't support sidebar.